### DISCUSSION TITLE
issue with https only mode in firefox and http-only websites

### DISCUSSION THREAD

**Discussion by rezad1393** [Purfview/IMDb-Scout-Mod#57] [Q&A] [answered]
when I enable https only mode in firefox this addon doesnt work with http websites even though I can open then in firefox and then add then to open in http mode exeption
is this a firefox but (feature?) it seems that when I enable that method then the scripts connections get converted to https even for website that I can open in http with firefox https mode enabled.

I am familiar with firefox internal to know that happens with scripts that user add and how they are treated, but I have seen issues like this before with first party isolation causing issues with sites that need login.

are you familiar with that?
any plan for future so that scripts like this work witha fpi-enabled and https-enabled firefox?

**Comment by Purfview** [comment, marked as answer]
If you don't want to use "https only mode" then why you enable it?
About FPI issues you should ask Firefox devs.
